.eslintrc

```diff
@@ -72,6 +72,12 @@
     "jsx-a11y/no-static-element-interactions": "off",
     "no-console": "error",
     "no-plusplus": "off",
+    "no-restricted-syntax": [
+      'error',
+      'ForInStatement',
```

Comment:
This was here so we'd explicitly go through and find instances where we were using `for... in` loops and remove them eventually. I know it's annoying to write it all the time but I thought we'd agree it would be an eventual code quality thing so writing them out would make it more obvious we don't encourage it and easy to grok for when replacing.

https://github.com/mozilla/addons-frontend/pull/1379#issuecomment-261549708

If we want to just say "okay we'll use `for... in`" I guess that's fine but it seems at odds with our general non-mutating stuff style. I'd be happy to stop using it eventually 😄